### DISCUSSION TITLE
Error use function 'set'

### DISCUSSION THREAD

**Discussion by chenjiyuan13** [yalmip/YALMIP#1034] [General discussion]
Dear developers: 
I face a problem when i use Yalmip toolbox.
The commend run like 
constraint_ml = set(cone(zz,y2));
, and an error comes. Could please help me to solve this problem.
"SET has been considered obsolete for many years, and the time has come...
Update your code. https://yalmip.github.io/tutorial/basics/
error use constraint/set 
Obsolete command"

**Comment by johanlofberg** [comment]
As it says, SET is obsolete and you will have to remove it

https://yalmip.github.io/tutorial/basics/
https://yalmip.github.io/command/cone/
